### PR TITLE
test: upgrade kubernetes versions to 1.25.0/1.26.0

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR updates k8s verison in smoke tests

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan test

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test matrix

**Comment by weizhouapache** [comment]
@blueorangutan test matrix

**Comment by weizhouapache** [comment]
@blueorangutan test

**Comment by weizhouapache** [comment]
@blueorangutan test matrix

File: test/integration/smoke/test_kubernetes_clusters.py [hunk 1]
```diff
@@ -68,6 +68,9 @@
     "tier_netmask": "255.255.255.0"
 }
 
+K8S_VERSION_v1 = "1.25.0"
+K8S_VERSION_v2 = "1.26.0"
```
Comment (DaanHoogland) [inline test/integration/smoke/test_kubernetes_clusters.py:72]: should we get this from test_data?

File: test/integration/smoke/test_kubernetes_supported_versions.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def setUpClass(cls):
         cls.services = cls.testClient.getParsedTestDataConfig()
         cls.zone = get_zone(cls.apiclient, cls.testClient.getZoneForTests())
         cls.mgtSvrDetails = cls.config.__dict__["mgtSvr"][0].__dict__
-        cls.kubernetes_version_iso_url = 'http://download.cloudstack.org/cks/setup-1.24.0.iso'
+        cls.kubernetes_version_iso_url = 'http://download.cloudstack.org/cks/setup-1.26.0.iso'
```
Comment (DaanHoogland) [inline test/integration/smoke/test_kubernetes_supported_versions.py:48]: should we get this from test_data as well?

File: test/integration/smoke/test_kubernetes_clusters.py (hunk 1 above)
Comment (weizhouapache) [reply test/integration/smoke/test_kubernetes_clusters.py:72]: I'd like to keep this in the test script file.

File: test/integration/smoke/test_kubernetes_supported_versions.py (hunk 2 above)
Comment (weizhouapache) [reply test/integration/smoke/test_kubernetes_supported_versions.py:48]: this should be read from test_data.  I will change it

Comment (weizhouapache) [reply test/integration/smoke/test_kubernetes_supported_versions.py]: done

File: test/integration/smoke/test_kubernetes_clusters.py (hunk 1 above)
Comment (weizhouapache) [reply test/integration/smoke/test_kubernetes_clusters.py]: this has been moved to test_data.py as well

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test

**Review by DaanHoogland — APPROVED** [review]
LGTM, a smoke test run should do for validating this. Would you agree @weizhouapache ?

**Comment by weizhouapache** [comment]
> LGTM, a smoke test run should do for validating this. Would you agree @weizhouapache ?

yes @DaanHoogland .
let's wait for the test results. (I tested manually on marvin node, it looks ok)

**Pull request merged**